net/bakedroots/bakedroots_test.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package bakedroots
+
+import "testing"
+
+func TestBakedInRoots(t *testing.T) {
+	ResetForTest(t, nil)
+	p := Get()
+	got := p.Subjects()
+	if len(got) != 1 {
+		t.Errorf("subjects = %v; want 1", len(got))
+	}
```

Comment:
optional: validate the actual subject names

Reply:
I don't see how to (easily?) with the Go API.

I'd need to do my own DER decoding? There's no constructor for just the DER-encoded subject?